### ISSUE TITLE
Internal spec review notes

### ISSUE THREAD

**Issue by johnluther** [w3c/webmediaguidelines#39]
# Content comments:

- Seems to be something missing from the sentence "it will not direction to device manufactures or User Agent implementers"
- In "Content Generation" section of Streaming Overview, the way its written implies that content will always be DRM'd when in fact DRM is optional.
- In the "Content Playback" section of Streaming Overview, same DRM comment, we should make it clear that not all content in these apps will be DRM'd. For example, make step one something like, "If the content is encrypted with DRM, the client uses the DRM license URL in the manifest (or media stream header) to request a secure key to enable decoding of the media."
- I found the "Content Generation" section of VOD very hard to parse. What does "prioritize density over quality" mean? Do we really need a discussion of video compression here? Also, is VoD source really still "typically" from tape in 2017? More likely a mezzanine file. Shouldn't the "client side restrictions" mentioned here be in the Playback section?
- "Trick Play" should probably be in the glossary.
- "src parameter of the of the video element" should clarify this refers to the HTML <video> element.
- The "HDCP security requirement for HDMI" section is overly opinionated about DRM. IMO phrases like "illicit market" and "potential pirate" aren't appropriate for a technical document. I would remove the whole second paragraph, it isn't needed. Also, if this is a best practices/guidance document, the recommended practice is unclear here. It seems to be "always enable HDCP to prevent piracy," which again, is not appropriate in a spec. Shouldn't it be something like, "evaluate whether your apps will need to support HDCP"? 
- "Watermarking" again, what is the best practice here?
- In "Unique device Identifier", the phrase "you need to be able to identify that device uniquely" should be "you MAY need to identify that device uniquely", since in many use cases a unique ID is  not required.
- "Like most DRM Widevine..." what is the point of this paragraph? IMO it is too low-level and unnecessary.
- "Currently Microsoft’s PlayReady dominates the market" is overly opinionated and unnecessary. 
- "video or audio media element" make clearer that this is the HTML <video> or <audio> element
- section 6 has a lot of broken sentence fragments. Maybe a ReSpec markup problem?
- "Original Content is normally delivered to the service provider as a file with near lossless compression." Might be worth mentioning that these are the mezzanine files referenced later.

# Typos and other editorial

- Not all headings are numbered. IMO they should be since many of the sections have the same title (e.g., "Content Generation"), which makes it difficult to reference them.
- "The content provider supply's", I think that should be "supplies"?
- Slash after the word "origin" seems like a typo
- "In HLS if there is a type is VOD"
- "10pc" should be 10%
- "The content deliver through a CDN" should that be "delivered"?
- "buffer windows" should be window
- " live window and event beyond it" should that be 'even'?
- In 2.4, "Unfortuantely" spelled wrong
- "Within an the application layer"
- "Device Type" section is missing proper paragraph breaks.
- "windows, mac, android" should be capitalized
- "classic pc /browser" should be PC browser
- "stb" is inconsistently capitalized
- "Advertisements" is inconsistently capitalized
- "key value pairs" should be key-value or key/value
- "manifest" is inconsistently capitalized
- "use-case" is inconsistently hyphenated
- "Application" is inconsistently capitalized
- "Mezzanine" is inconsistently capitalized
- "Encoding" is inconsistently capitalized
- "mp4" is inconsistently capitalized
- "VoD" is inconsistently capitalized
- "Progressive Web apps" is inconsistently capitalized
- "not exceed bandwidth availability" needs a period
- "In web terms. " seems to be an orphaned clause

**Comment by joelkorpi** [comment]
Thanks for the detailed review, John! I'm fixing these issues now...

**Comment by joelkorpi** [comment]
I addressed all of these issues in the latest merge! Please review and let me know if I missed anything @johnluther
